src/libraries/Directory.Build.props

```diff
@@ -23,12 +23,12 @@
     <IsSourceProject>$([System.Text.RegularExpressions.Regex]::IsMatch($(MSBuildProjectDirectory), 'src%24'))</IsSourceProject>
     <LanguageTargets Condition="'$(MSBuildProjectExtension)' == '.depproj'">$(RepositoryEngineeringDir)depProj.common.targets</LanguageTargets>
     <RuntimePropsFile>$([MSBuild]::NormalizePath('$(ArtifactsDir)', 'tmp', '$(Configuration)', 'RuntimeOS.props'))</RuntimePropsFile>
-    <RuntimeGraph>$(LibrariesProjectRoot)\OSGroups.json</RuntimeGraph>
+    <RuntimeGraph>$(LibrariesProjectRoot)\pkg\Microsoft.NETCore.Platforms\runtime.json</RuntimeGraph>
```

Comment:
Can you use https://github.com/dotnet/runtime/blob/33a6ed57e354f270d86ac4b337c07691298093f2/Directory.Build.props#L68